### ISSUE TITLE
[Bug] Control.SearchBarTextField causes crash on iOS 12

### ISSUE THREAD

**Issue by wojciech-kulik** [xamarin/Xamarin.Forms#8255]
### Steps to Reproduce

1. Create custom renderer inheriting from`SearchBarRenderer`
```
protected override void OnElementChanged(ElementChangedEventArgs<SearchBar> e)
{
    base.OnElementChanged(e);
        if (Control != null)
        {
            Control.SearchTextField.AutocorrectionType = UITextAutocorrectionType.No;
        }
}
```

### Expected Behavior

Property should work or there should be warning that it's not available below iOS 13.

### Actual Behavior

It causes crash on iOS 12 `unrecognized selector sent to instance`

### Basic Information

- Last known good version: -
- IDE: Visual Studio for Mac 8.3.4 (#8)
- Platform Target Frameworks:
  - iOS:  13.1
- Nuget Packages: 4.2.0.848062
- Affected Devices: iPhone/iPad with iOS 12.

**Comment by hartez** [comment]
We can't catch and warn about this directly in Forms - you'd need to implement your own version check. However, we could make it easier to access the SearchTextField when writing a custom renderer - see #8256.

**Comment by wojciech-kulik** [comment]
So how do I know then which property can crash and which can't? 

In this case, I've just used `Control.SearchTextField.AutocorrectionType = UITextAutocorrectionType.No;` not expecting any risk here and after uploading to AppStore I started receiving crashes from Firebase and only then I figured out that there should be a version check and for older iOS versions I used this code: `Control.AutocorrectionType = UITextAutocorrectionType.No;`. I think it should at least not crash.
